packages/caliper-core/lib/worker/tx-observers/prometheus-manager-tx-observer.js

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
This isn't prometheus specific at this point so I think we could just call this ManagerTxObserver and the file manager-tx-observer,js

Reply:
I have named it this way because it would be inline with the transaction monitor module in the config file. In the existing codebase `module: prometheus-push` is bound to a `PrometheusPushTxObserver` and `module: prometheus` is bound to a `PrometheusTxObserver`, etc., so `PrometheusManagerTxObserver` would be the equivalent for the planned `module: prometheus-manager` config property: https://github.com/hyperledger/caliper/blob/4aa36ea91e21966d4aac65d04a36720ee93ca36d/packages/caliper-core/lib/worker/tx-observers/tx-observer-dispatch.js#L21-L25

Should I perform the refactor to `ManagerTxObserver`?